roles/ftp/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+- name: make ftp dir
+  file:
+    name: "{{ item }}"
+    state: directory
+  with_items:
+    - "{{ ftp_dir }}"
+    - "{{ ftp_dir }}/data"
```

Comment:
I need to take a look at the default FS layout of the conda-channel-resource, as well as the existing nginx server, to update this accordingly.

Reply:
The easiest solution would be to set up a symbolic link, so once we sort out what needs to be linked where, we can add a step to this set of tasks.